### DISCUSSION TITLE
Conbee II or Sonoff USB Dongle Plus

### DISCUSSION THREAD

**Discussion by jmachta** [Koenkk/zigbee2mqtt#16307] [Devices]
Guys, I would like to hear your experience about zigbee dongles. I am using for almost a year a Conbee II dongle. After CC2531 bare board this was a massive improvement in reliability. However, I have a Sonoff Zigbee 3.0 USB Dongle Plus, which seems to be more robust (mechanically), plus has a bigger antenna => might have better range? Conbee II works mostly well, but sometimes my devices drop messages, so that e.g. brightness of the bulb is not set from time to time... 

My question is: should I try to swap the dongles and use Sonoff one instead of Conbee II? Does anybody have such experience? After my last attempts with swapping dongles (I just replaced one conbee with another one) I had to repeat binding of all IKEA devices, same with Aqara and Sonoff. The only that survived this were Phillips HUE bulbs. 

So, I am asking first if this is worth of the effort or if I should just keep the Sonoff dongle as a backup? Thank you a lot for any suggestions.

**Comment by jakubch82** [comment]
Definitely worth it. I was using Conbee II from 2021 to mid 2022. It was rock solid. But then it started to giving me random genOnOff errors. I couldn't find the cause. My Conbee was on the latest stable firmware. The situation was even worse If I wanted send multiple commands to devices at once. I suppouse that newer versions of Home Assistant and Zigbee2mqtt messed up something with communication at hardware layer - maybe bad handling messages queues. Then I took a shot and switch to Sonoff USB Dongle P. For me it was it. No more errors. No more lost commands. My zigbee network consist on 40+ devices - Ikea switches, Ikea routers, Aqara and Philips Hue mostly motion sensors (I'm using Ikea outlets as routers because they work well with Aqara devices). Sonoff USB Dongle P is very snappy. It turns On/Off about 20 devices at once without any complaints :) I'm using it since mid 2022 til now and I didn't get any single error. I'm working on Home Assistant 2022.11.5, zigbee2mqtt 1.28.4-1, Sonoff USB Dongle P with zStack3x0 20220219 firmware. Now I'm scared to update them to newer version because it works so smooth and well. I'm constantly reading forums, git hub issues and it worries me a bit that many other users has so many issues with thoes updates. Guess time will tell but for now I'm not gonna update to newer versions.

**Reply by jmachta** [reply]
Thank you for the response. I decided to follow your suggestion and gave it a try. It was a nightmare to force some of the Ikea devices to bind to the new network, but after some hours I made it. And voilla, I need to confirm, the improvement is really there. 

Network scan is much faster, takes few tens of seconds, no random lost messages or sporadic errors. With some devices I am now able to do actions which did not work before. E.g. on Aqara wall switches I was not able to flip LED behavior. Now these strange issues are gone. Plus, complete network seems much more reliable, I can command several devices at once. Before, I had to interleave commands with sleeps. 

So, once more, thanks!
